starter-code/bonus-testing-part-two.js

```diff
@@ -33,6 +42,22 @@ function testIsNumber() {
   );
 }
 
+function testProducesAnswer() {
+  assert(
+    tooHungryDay <= mealsPerDay.length,
+    'The day the tiger gets too hungry falls within the limits of the array',
+    'tooHungryDay does not occur within the limits of this array'
+  );
+}
+
+function testHasValidNumbers(){
+  assert(
+    tooHungryDay > 0,
+    'Day is greater than 0',
+    'Day is less than 0'
+  );
+}
+
```

Comment:
A good exception to catch here - can you think of a way to combine the conditional part of these two functions into one line?